### ISSUE TITLE
Female Fragment Hat Raichu Sprites Missing

### ISSUE THREAD

**Issue by PaulGolting** [ZeChrales/PogoAssets#13]
Hello! This is my first time ever submitting an "issue" on GitHub. The last ZIP that I downloaded of all the PoGo assets just a few days ago does not have these Sprites: 
-Female Fragment Hat Raichu
-Shiny Female Fragment Hat Raichu 

Also, I noticed a bunch of duplicate Alolan Raichu Sprites, both normal & Shiny. Attached are some screenshots of the files in the ZIP.

Anyways, just thought I would mention it. Not sure what happened or if maybe I'm totally just missing it. Thanks for all your work with this! 

![screen shot 2018-10-11 at 3 32 27 pm](https://user-images.githubusercontent.com/44073992/46837666-10b08a00-cd6b-11e8-9114-dfd7082a993e.png)
<img width="1000" alt="screen shot 2018-10-11 at 3 33 09 pm" src="https://user-images.githubusercontent.com/44073992/46837667-10b08a00-cd6b-11e8-8f60-b31bcf07a2c6.png">

**Comment by PaulGolting** [comment]
@ZeChrales - Hello! First of all, thank you for all your hard work providing such a wonderful resource to the PoGo community.  

Now, I've been seeing issues being replied to by multiple people and issues being closed but I haven't heard anything about this issue. I'm not sure what Discord server you are on, so I figured I would just message here. Not sure if this issue with the Fragment Sprites is valid or if it's just me. 

Anyways, thanks for your time!

**Comment by ryankane** [comment]
The female sprite was never committed.

**Comment by PaulGolting** [comment]
> The female sprite was never committed.

Hello @ryankane ! Thanks for commenting. What exactly do you mean by that?

**Comment by bdaubry** [comment]
@ryankane @PaulGolting it's not that it was never committed, it doesn't exist in network traffic. Niantic didn't put it on the server, basically. Check out this [imgur album](https://imgur.com/a/BL5M9mq). If you notice, both of the fragment hat sprites show as male. Yet, the one with 1911CP is definitely female. They don't have the right sprite on the server, so they both show as male in your storage. If you look at the witch hat Raichu's, you can see that the correct sprites were added. 

A similar issue happens with shiny Male Luxio. The sprite shows the non-shiny version, even though it is shiny. I can't show any screenshots of it, as I don't have one, but it has been posted about on TheSilphRoad. However, this issue differs because there actually is a sprite uploaded with the correct filename, but it has the non-shiny form. Basically, there are no differences between the regular and shiny sprite files. 

For the Alolan Raichu, the naming scheme will give a bit of a hint. The filenames end with 3 numbers. 026, the Pokedex number, 00, 01, or 61, the gender/Alolan reference number, and then 00 through 06, the form reference number. When Alolans were added, the gender numbers were changed with a leading 6 instead of a 0. However, since no Alolan Pokemon show gender differences, the Alolan "form" is always 61. For some reason on Niantics end, they always add a new Alolan form for every hat form that comes out. However, since they aren't available with hats, they are all the same sprite. 

Anyway, long story short, all the icon filenames follow the same format. `pokemon_icon_pokedexNumber[_gender][_form].png`, where anything in [] is 'if applicable'

**Comment by bdaubry** [comment]
[Here's a link to the issue with a screenshot of the Luxio shiny sprite being wrong](https://github.com/ZeChrales/PogoAssets/issues/16)

**Comment by PaulGolting** [comment]
@bdaubry - Thank you so much for your reply! I was thinking that was the case with the Female Frag Hat Sprites - that Niantic messed up with the Sprite files on their servers. However, I wasn't sure if that was what actually happened. With you mentioning it and proving it with the screenshots, that is all that I needed! 

As for the Alolan Raichu, I appreciate you providing that explanation - it makes total sense! However, is that your speculation about what happened, or is that something you know for sure is what happened? Just wondering. Either way, it seems like that is what happened. 

Again, thanks for your detailed comment with all your info and help!

**Comment by bdaubry** [comment]
@PaulGolting ZeChrales only adds stuff here that he is able to pull from network traffic, so there is a un-hatted Alolan Raichu sprite and image file for every hatted Raichu variant. Why Niantic does it that way, I can't honestly say (maybe they intend to add Alolan Raichu with a hat in the future, and all it would take would be updating the sprite assets) or maybe it's just a consequence of the way they do forms on the back end.

**Comment by PaulGolting** [comment]
@ryankane - Oh cool! Thanks for the Sprites and for the info you provided!

**Comment by ryankane** [comment]
This issue should now be marked CLOSED, since the sprites were updates and added.

See commit "raichu update" f0b495a02c8673d845c3850dc1caf3e5c0b29812

**Comment by PaulGolting** [comment]
@ryankane - Awesome! I just got the new files! Glad that it was fixed! :)
